oxd-gen-client/src/test/java/io/swagger/client/api/RegisterSiteTest.java

```diff
@@ -101,7 +102,9 @@ public static RegisterSiteResponseData registerSite(DevelopersApi apiClient,
                 GrantType.OXAUTH_UMA_TICKET.getValue(),
                 GrantType.CLIENT_CREDENTIALS.getValue()));
 
-        final RegisterSiteResponse resp = apiClient.registerSite(params);
+        final ApiResponse<RegisterSiteResponse> regApiResponse = apiClient.registerSiteWithHttpInfo(params);
+        final RegisterSiteResponse resp = regApiResponse.getData();
+        assertTrue(regApiResponse.getStatusCode() == 200);
```

Comment:
@shoebkhan09 This code is redundant. If you get `RegisterSiteResponse` then it means that status code is 200, otherwise you will get `ApiException`.